src/payment/CardHolderInformation.jsx

```diff
@@ -137,16 +143,17 @@ class CardHolderInformation extends React.Component {
   }
 }
 
-CardHolderInformation.propTypes = {
+CardHolderInformationComponent.propTypes = {
+  dispatch: PropTypes.func.isRequired,
   intl: intlShape.isRequired,
   countryOptions: PropTypes.arrayOf(PropTypes.shape({
     label: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   })),
 };
 
-CardHolderInformation.defaultProps = {
+CardHolderInformationComponent.defaultProps = {
   countryOptions: [],
 };
 
-export default connect(countryOptionsSelector)(injectIntl(CardHolderInformation));
+export default connect(countryOptionsSelector)(injectIntl(CardHolderInformationComponent));
```

Comment:
If you add:

```
{
  clearFields,
}
```
as a second argument to the `connect` function, you can get rid of the `dispatch` prop and call `this.props.clearFields` directly.  Connect will wrap any functions provided in that object in dispatch automagically for you.